python_frank_energie/frank_energie.py

```
@@ -1441,128 +1501,186 @@ async def smart_battery_sessions(
+                        },
+                        {
+                            "cumulativeTradingResult": 2.97558131215891,
+                            "date": "2025-03-05",
+                            "tradingResult": 0.3774874975,
+                        },
+                    ],
+                    "totalTradingResult": 55.14711599931087,
+                }
+            }
+        }
+        mock_response = {
```

Ruff (0.12.2)

Local variable `mock_response` is assigned to but never used

Remove assignment to unused variable `mock_response`

(F841)